Unable to resolve package

### Description

When trying to do wrangler publish, I get this error:

```
✘ [ERROR] Could not resolve "cloudflare-discord-bot"

    src/Discord/index.js:6:7:
      6 │ } from "cloudflare-discord-bot"
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "cloudflare-discord-bot" as external to exclude it from
  the bundle, which will remove this error.
```

This is not the case with `@glenstack/cf-workers-discord-bot` package.

### Reproducibility

- Using the readme example.
- Using wrangler beta.
- Worker module type.
- `wrangler publish`

### Version

0.0.1 (latest)

### What browsers are you seeing the problem on?

Chrome

### Log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "author": "alvgaona@gmail.com",
   "license": "Apache License 2.0",
   "main": "dist/index.js",
-  "types": "dist",
+  "types": "dist/index.d.ts",
   "files": [
     "/dist"
   ],

